sampleAppKotlin/src/main/res/values/styles.xml

```diff
@@ -8,4 +8,15 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="VirtusizeDesignSystemFlow">
+        <item name="android:paddingStart">@dimen/virtusize_16dp</item>
+        <item name="android:paddingEnd">@dimen/virtusize_16dp</item>
+        <item name="flow_horizontalBias">0</item>
```

Comment:
Might it be `flowHorizontalBias `?

Reply:
This is a default attribute from the official Android library